release-notes/7.0/supported-os.md

```diff
@@ -37,6 +37,12 @@ OS                                    | Version               | Architectures
 [SUSE Enterprise Linux (SLES)][SLES]  | 12 SP2+               | x64               | [SUSE][SLES-lifecycle]
 [Ubuntu][Ubuntu]                      | 16.04, 18.04, 20.04+  | x64, Arm64, Arm32 | [Ubuntu][Ubuntu-lifecycle]
 
+Other distros are supported with best effort if their libc version is at least the minimum supported libc version:
+
+- x64: [glibc][glibc] 2.17 (from CentOS 7)
+- Arm64: [glibc][glibc] 2.23 (from Ubuntu 16.04)
```

Comment:
After the upcoming change, this will be glibc 2.27 for arm64 and arm32 (from Ubuntu 18.04). So Ubuntu 16.04 won't be supported anymore for arm64 and arm32.

Reply:
Oh right, that is what you told me. Does the suggestion look good?

```suggestion
- Arm32/64: [glibc][glibc] 2.27 (from Ubuntu 18.04)
```